src/__tests__/index.test.js

```diff
@@ -67,6 +69,35 @@ describe('Get Osmose errors', () => {
   });
 });
 
+describe('Update Osmose errors', async () => {
+  const language = 'fr';
+  const osmose = new OsmoseRequest({ language });
+  const errors = await osmose.fetchErrors({ item: 8120 });
```

Comment:
... So you have to fetch the errors in each `it` (not a perf problem since we have mocks)